fish_prompt.fish

```diff
@@ -119,7 +120,8 @@ end
 function __bobthefish_hg_project_dir -S -d 'Print the current hg project base directory'
   [ "$theme_display_hg" = 'yes' ]; or return
   set -l d $PWD
-  while not [ $d = / ]
+  set stop_on_dirs / $theme_hg_ignored_dirs
+  while not contains $d $stop_on_dirs
```

Comment:
I don't think we need variable assignment here. Just

```
while not contains $d / $theme_hg_ignored_dirs
  ...
```

should do the trick?